atlasdb-api/src/main/java/com/palantir/atlasdb/keyvalue/api/ColumnSelection.java

```diff
@@ -61,7 +61,7 @@ public static ColumnSelection valueOf(String serialized) {
     @Override
     public String toString() {
         if (selectedColumns == null) {
-            return "";
+            return "*";
```

Comment:
I'm a little hesitant to change this, and leaning toward either revert or update `ColumnSelection.valueOf` to explicitly handle this case. I don't see any callers of `com.palantir.atlasdb.keyvalue.api.ColumnSelection#valueOf` in any projects (even tests), so my sense is that `valueOf` is dead code.